model/src/main/kotlin/config/FileArchiverConfiguration.kt

```diff
@@ -33,14 +37,35 @@ data class FileArchiverConfiguration(
     /**
      * Configuration of the [FileStorage] used for archiving the files.
      */
-    val storage: FileStorageConfiguration
-)
+    val fileStorage: FileStorageConfiguration? = null,
 
+    /**
+     * Configuration of the [PostgresFileArchiverStorage] used for archiving the files.
+     */
+    val postgresStorage: PostgresStorageConfiguration? = null
+) {
+    init {
+        if ((fileStorage != null) && (postgresStorage != null)) {
+            log.warn { "At most one storage should be specified." }
```

Comment:
Why do we warn about this? Are the some technical risks when doing this, or is it just because it's usually not needed to configure multiple storage here? Can we make this clearer by rewording the warning?

Reply:
The warning is because only one storage can be used at the same time. I have reworded the message. I think we should replace the last sentence of the new message with always printing details about the used storage, but I don't want to do this now.